cmd/vela-worker/register.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package main
+
+import (
+	"net/http"
+
+	"github.com/go-vela/types/library"
+	"github.com/sirupsen/logrus"
+)
+
+// register is a helper function to register
+// the worker in the database, updating the item
+// if the worker already exists
+func (w *Worker) register(config *library.Worker) {
```

Comment:
I think we should have this function return an `error` 👍 

```suggestion
func (w *Worker) register(config *library.Worker) error {
```